rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/kafka/TbKafkaNodeConfiguration.java

```diff
@@ -23,6 +25,7 @@
 import java.util.Map;
 
 @Data
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Better add upgrade script and remove the fields from the configuration. Also, UI needs to be updated

Reply:
https://github.com/thingsboard/thingsboard/pull/12788

Reply:
@YevhenBondarenko, since you've added the upgrade script, we shouldn't ignore any properties. I recommend removing this annotation so that config parsing fails if unexpected properties are present.